content/en/docs/kubernetes/operator/automatic.md

```diff
@@ -361,11 +361,11 @@ spec:
 
 #### Excluding auto-instrumentation {#python-excluding-auto-instrumentation}
 
-By default the Python auto-instrumentation will detect the packages in your
-Python service and instrument anything it can. This makes instrumentation easy,
-but can result in too much or unwanted data. If there are any packages you do
-not want to instrument, you can set the `OTEL_PYTHON_DISABLED_INSTRUMENTATIONS`
-environment variable
+By default, the Python auto-instrumentation ships with
+[many instrumentation libraries]([https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/blob/main/docs/config.md#instrumentations](https://github.com/open-telemetry/opentelemetry-operator/blob/main/autoinstrumentation/python/requirements.txt)).
```

Comment:
here something went wrong, probably you only wanted to have the last URL?

```suggestion
[many instrumentation libraries](https://github.com/open-telemetry/opentelemetry-operator/blob/main/autoinstrumentation/python/requirements.txt).
```

Reply:
Thank you! Need to work on my copy/paste skills! 😁

Reply:
same, happens to me sooo often ;-)